src/Fixer/AbstractPhpUnitFixer.php

```diff
@@ -168,7 +168,7 @@ private static function isPreventedByAttribute(Tokens $tokens, int $index, array
 
         foreach (AttributeAnalyzer::collect($tokens, $index) as $attributeAnalysis) {
             foreach ($attributeAnalysis->getAttributes() as $attribute) {
-                if (\in_array(self::getFullyQualifiedName($tokens, $attribute['name']), $preventingAttributes, true)) {
+                if (\in_array(ltrim(self::getFullyQualifiedName($tokens, $attribute['name']), '\\'), $preventingAttributes, true)) {
```

Comment:
Shouldn't `getFullyQualifiedName` ltrim the leading backslash instead?

Reply:
Potayto, potahto - in here `ltrim` is written once, in `getFullyQualifiedName` would be a few times. https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8048 will replace this call anyway.